Projects/Profile/Sources/Views/Screens/EuroBonus/ChangeEuroBonusView.swift

```diff
@@ -42,37 +48,13 @@ private class ChangeEurobonusViewModel: ObservableObject {
             let data = try await self?.profileService.update(eurobonus: text)
             let store: ProfileStore = globalPresentableStoreContainer.get()
             store.send(.setEurobonusNumber(partnerData: data))
-            store.send(.openSuccessChangeEuroBonus)
-        }
-    }
-}
 
-extension ChangeEuroBonusView {
-    static var journey: some JourneyPresentation {
-        HostingJourney(
-            ProfileStore.self,
-            rootView: ChangeEuroBonusView(),
-            style: .detented(.scrollViewContentSize),
-            options: [.largeNavigationBar, .blurredBackground]
-        ) { action in
-            if case .openSuccessChangeEuroBonus = action {
-                SuccessScreen.journey(with: L10n.SasIntegration.eurobonusConnected)
-                    .onPresent {
-                        DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
-                            let store: ProfileStore = globalPresentableStoreContainer.get()
-                            store.send(.dismissChangeEuroBonus)
-                        }
-                    }
-            }
-        }
-        .configureTitle(L10n.SasIntegration.enterYourNumber)
-        .onAction(ProfileStore.self) { action in
-            if case .dismissChangeEuroBonus = action {
-                PopJourney()
-            }
+            /* TODO: SOMEHOW SET SUCESS HERE*/
+            //  router.push(EuroBonusRouterType.successChangeEuroBonus)
```

Comment:
@sladan-hedvig having some issues here on how to push success screen. Any ideas?